src/partition/repart.c

```diff
@@ -3916,6 +3924,153 @@ static int context_mangle_partitions(Context *context) {
         return 0;
 }
 
+static int split_name_printf(Partition *p) {
+        assert(p);
+
+        const Specifier table[] = {
+                { 't', specifier_id128, &p->type_uuid },
+                { 'u', specifier_id128, &p->new_uuid  },
```

Comment:
i have the suspicion people would want to use the assigned partition number soon, too. so that people could specify `SplitName=p%n` or so, to get the naming fdisk uses when showing contents of a disk image.

Reply:
So, yeah %n for the partition number and %T for the "friendly" name for a partition type, as reported by gpt_partition_type_uuid_to_string_harder() sounds super useful to me.